osquery/events/windows/etw/etw_publisher_processes.cpp

```diff
@@ -0,0 +1,552 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <ctime>
+
+#include <osquery/core/flags.h>
+#include <osquery/events/windows/etw/etw_publisher_processes.h>
+#include <osquery/utils/conversions/windows/strings.h>
+#include <osquery/utils/map_take.h>
+
+namespace osquery {
+
+FLAG(bool,
+     enable_etw_process_events,
```

Comment:
Let's make sure this flag changes along with the table name.

Reply:
Fixed